src/plugins/devtool.js

```diff
@@ -15,5 +20,10 @@ export default function devtoolPlugin (store) {
 
   store.subscribe((mutation, state) => {
     devtoolHook.emit('vuex:mutation', mutation, state)
+
+    // if buffer exists push mutation for later use
+    if (Array.isArray(store._devtoolBuffer)) {
```

Comment:
Why not just `if(store._devtoolBuffer) {`?

Reply:
Done